### ISSUE TITLE
CPU usage is inconsistent

### ISSUE THREAD

**Issue by MartinRumanek** [cockpit-project/cockpit#2016]
How it's possible?

usage is around 4 %
![home](http://i.imgur.com/DfJfFRv.png)

at the same time the cpu usage on the container page is much larger (I have 2 cores)
![containers](http://i.imgur.com/RrRXOiG.png)

**Comment by MartinRumanek** [comment]
Should I provide any further information?

**Comment by stefwalter** [comment]
I can't duplicate this here. But I'm having a hard time seeing any CPU information on the Containers page ... :S

**Comment by stefwalter** [comment]
Any chance you could update to a recent Cockpit version (such as 0.48 ... which you can get in Fedora 22)?

**Comment by MartinRumanek** [comment]
I've tested it on virtual machine..  As it turned out docker containers are not the problem. But open-vm-tools package is problematic. When I uninstalled it, problem was gone.

I've also tested Cockpit 0.48  (installed by `yum install bash --enablerepo rawhide`, dependencies are from Fedora 21) and now the title page is OK.:
![title](http://i.imgur.com/iaFI4LC.png)

However if I click on this graph, this is the result I get:
![cpu_detail](http://i.imgur.com/OUuS3Sg.png)

The system was loaded by command `cat /dev/zero > /dev/null`
